### PR TITLE
Add update guide

### PR DESCRIPTION
**Pull request by e-im**
Adds a guide for updating Paper.

Please don't merge this yet. Pretty much blocked by PaperMC/Paper#6575

### CODE REVIEW

File: source/server/update.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Just like it’s important to update Paper, it’s equally important to keep
+plugins up to date. You never know what plugin authors may be working
+on, be it bugfixes or new features.
+
+A little known feature of Paper servers is the ``update`` folder. Here’s
+how you use it.
+
+1. Create a folder called ``update`` within the ``plugins`` folder.
+2. One by one, download plugins you would like to update and put them
```
Comment (NoahvdAa) [inline source/server/update.rst:46]: Might be worth noting here that this currently (until PaperMC/Paper#6575 is merged) will only work if the plugin jar has the same name as the previous version.

Comment (sulu5890) [reply source/server/update.rst:46]: I plan to wait on that before merging this, as the update folder is a lot less user-friendly without it.

File: source/server/update.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    it’s never a good idea to replace any JAR in a running server, be that
+    plugins, or Paper itself.
+
+Step 1. Backup
+--------------
+
+This is the most important step, and yet the most frequently skipped.
+While it is unlikely that updating Paper itself will cause any issues
+requiring you to restore from a backup, plugin malfunctions or other
+accidents might! Updating is a great time to work in a backup. Having
```
Comment (MiniDigger) [inline source/server/update.rst:27]:
```suggestion
accidents might! Updating is a great time to work on a backup plan. Having
```

**Review by MiniDigger — APPROVED** [review]
I would also add something like, consider testing on a test/local server first.

but this is looking really good already!

**Comment by e-im** [comment]
Included in https://github.com/PaperMC/docs

**Pull request closed without merging**